### PR TITLE
Change help text to use source/target terminology in multicluster CLI

### PR DESCRIPTION
**Pull request by kleimkuhler**
Change terminology from local/remote to source/target in `multicluster` CLI help
text.

This does not change any variable, function, struct, or field names since
testing is still improving.

Relevant issue: #4480

Signed-off-by: Kevin Leimkuhler <kevin@kleimkuhler.com>

### CODE REVIEW

File: cli/cmd/multicluster.go [hunk 1]
```diff
@@ -151,7 +151,7 @@ func buildMulticlusterInstallValues(opts *multiclusterInstallOptions) (*multiclu
 	}
 
 	if opts.gatewayProbePort == defaults.GatewayLocalProbePort {
-		return nil, fmt.Errorf("The probe port needs to be different from %d which is the local probe port", opts.gatewayProbePort)
+		return nil, fmt.Errorf("The probe port needs to be different from %d which is the source probe port", opts.gatewayProbePort)
```
Comment (adleong) [inline cli/cmd/multicluster.go:154]: Hmmmm... what about "which is the multicluster probe port"?

File: cli/cmd/multicluster.go [hunk 2]
```diff
@@ -269,7 +269,7 @@ func newAllowCommand() *cobra.Command {
 
 	cmd.Flags().StringVar(&opts.namespace, "namespace", defaultMulticlusterNamespace, "The destination namespace for the service account.")
 	cmd.Flags().BoolVar(&opts.ignoreCluster, "ignore-cluster", false, "Ignore cluster configuration")
-	cmd.Flags().StringVar(&opts.serviceAccountName, "service-account-name", "", "The name of the remote access service account")
+	cmd.Flags().StringVar(&opts.serviceAccountName, "service-account-name", "", "The name of the target access service account")
```
Comment (adleong) [inline cli/cmd/multicluster.go:272]: Again, what do you think of "The name of the multicluster access service account"?

File: cli/cmd/multicluster.go [hunk 3]
```diff
@@ -364,13 +364,13 @@ func newMulticlusterInstallCommand() *cobra.Command {
...
 	cmd.Flags().StringVar(&options.logLevel, "log-level", options.logLevel, "Log level for the Multicluster components")
 	cmd.Flags().StringVar(&options.gatewayNginxImage, "gateway-nginx-image", options.gatewayNginxImage, "The nginx image to be used")
 	cmd.Flags().StringVar(&options.gatewayNginxVersion, "gateway-nginx-image-version", options.gatewayNginxVersion, "The version of nginx to be used")
 	cmd.Flags().StringVarP(&options.controlPlaneVersion, "control-plane-version", "", options.controlPlaneVersion, "(Development) Tag to be used for the control plane component images")
 	cmd.Flags().StringVar(&options.dockerRegistry, "registry", options.dockerRegistry, "Docker registry to pull images from")
-	cmd.Flags().BoolVar(&options.remoteMirrorCredentials, "remote-mirror-credentials", options.remoteMirrorCredentials, "Whether to install the default remote access service account")
+	cmd.Flags().BoolVar(&options.remoteMirrorCredentials, "target-mirror-credentials", options.remoteMirrorCredentials, "Whether to install the default target access service account")
```
Comment (adleong) [inline cli/cmd/multicluster.go:373]: What about just `service-mirror-credentials` and "Whether to install the service account which can be used by service mirror components in source clusters to discover exported services"

File: cli/cmd/multicluster.go [hunk 4]
```diff
@@ -707,7 +707,7 @@ func newExportServiceCommand() *cobra.Command {
 
 	cmd := &cobra.Command{
 		Use:   "export-service",
-		Short: "Exposes a remote service to be mirrored",
+		Short: "Exposes a service to be mirrored",
```
Comment (kleimkuhler) [inline cli/cmd/multicluster.go:710]: I think saying less is better in this description, but I could change it to `Exposes a target service to be mirrored`.

**Pull request merged**